bigchaindb/common/schema/vote.yaml

```diff
@@ -0,0 +1,69 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+id: "http://www.bigchaindb.com/schema/vote.json"
+type: object
+additionalProperties: false
+title: Vote Schema
+description: |
+    A Vote is an endorsement of a Block (identified by a hash) by
+    a node (identified by a public key).
+
+    The outer Vote object contains the details of the vote being made
+    as well as the signature and identifying information of the node
+    passing the vote.
+required:
+- node_pubkey
+- signature
+- vote
+properties:
+  node_pubkey:
+    type: "string"
+    pattern: "[1-9a-zA-Z^OIl]{43,44}"
+    description: |
+        Ed25519 public key identifying the voting node.
+  signature:
+    type: "string"
+    pattern: "[1-9a-zA-Z^OIl]{86,88}"
+    description:
+        Ed25519 signature of the ``vote`` object.
+  vote:
+    type: "object"
+    additionalProperties: false
+    description: |
+        Vote details to be signed.
+    required:
+    - invalid_reason
+    - is_block_valid
+    - previous_block
+    - voting_for_block
+    - timestamp
+    properties:
+      previous_block:
+        "$ref": "#/definitions/sha3_hexdigest"
+        description: |
+          Sha3 identifier of the block that preceeds the block being voted on.
```

Comment:
In this case, I think it would be okay to say it's "The id of the block that the node considers to be the block right before the block being voted on. In principle, another node might think a different block is the previous block."